docs/topics/proc_specify-a-custom-logging-framework-vertx.adoc

```diff
@@ -19,5 +19,5 @@ When implementing logging using a custom library, ensure that the relevant `Log4
 +
 [CAUTION]
 --
-Notice that the provided delegate for Log4J 1 does not support parameterized message. The delegate for Log4J 2 uses the `{}` syntax like the SLF4J delegate. JUL delegate uses the `\{x}` syntax.
+Notice that the provided delegate for Log4J 1 does not support parameterized message. The delegate for Log4J 2 uses the `{}` syntax like the SLF4J delegate. `java.util.logging` delegate uses the `\{x}` syntax.
```

Comment:
I noticed this before, but this `\{x}` looks suspicious. Do you have a link where this is described?

Reply:
The backslash is no part of the code example. it was to escape the `{x}` character group. Upon closer inspection it does exactly nothing. So I'm removing it:)